queries.sql

```diff
@@ -117,4 +117,84 @@ FROM owners o
 LEFT JOIN animals a ON o.id = a.owner_id
 GROUP BY o.full_name
 ORDER BY total_animals DESC
-LIMIT 1;
\ No newline at end of file
+LIMIT 1;
+
+
+
+-- Questions 
+-- Who was the last animal seen by William Tatcher?
+SELECT a.name 
+FROM animals a 
+JOIN visits v ON a.id = v.animals_id
+JOIN vets vet ON v.animals_id=vet.id
+WHERE vet.name = 'Vet William Tatcher '
+ORDER BY v.visits_date;
+-- How many different animals did Stephanie Mendez see?
+SELECT COUNT(*)
+FROM vets v 
+JOIN visits vi ON v.id = vi.vets_id
+WHERE v.name = 'Vet Stephanie Mendez ';
+-- List all vets and their specialties, including vets with no specialties.
+SELECT v.name 
+FROM vets v 
+LEFT JOIN specializations s ON v.id = s.vet_id
+LEFT JOIN species ss ON ss.id = s.species_id
+GROUP BY v.name, v.id;
```

Comment:
Good but the question here is to return vets data and their specialities and the query you provided returns only name and id , you can consider this solution and also add a group by clause to return each name with their respective specialities
`  SELECT vets.*, species.name
  FROM vets
  LEFT JOIN specializations ON vets.id = specializations.vet_id
  LEFT JOIN species ON specializations.species_id = species.id;
`